test/jit/test_unsupported_ops.py

```diff
@@ -48,11 +48,13 @@ def test_tensor_options_behavior_mismatch(self):
 
         # Complete issue and set of ops is https://github.com/pytorch/pytorch/issues/30763
         # only testing one here because they should be fixed all at once
+        # ezyang: But actually, I handled all of the _like overloads first
```

Comment:
In this or a follow up PR or PR later in the stack mind removing the `_like` overloads from https://github.com/pytorch/pytorch/blob/master/docs/source/jit_unsupported.rst#ops-with-divergent-schemas-between-torch--python ?